rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/BakeManifestService.java

```diff
@@ -63,6 +71,25 @@ public BakeManifestService(JobExecutor jobExecutor) {
     if (bakeStatus.getResult() != BakeStatus.Result.SUCCESS) {
       throw new IllegalStateException("Bake failed: " + bakeStatus.getLogsContent());
     }
-    return bakeStatus.getOutputContent().getBytes();
+    return removeTestsDirectoryTemplates(bakeStatus.getOutputContent()).getBytes();
+  }
+
+  public String removeTestsDirectoryTemplates(String input) {
```

Comment:
I think this probably makes more sense in `HelmBakeManifestService` since test manifests are only a concept within Helm.

Reply:
Moved to `HelmBakeManifestService `